docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,138 @@
+# Getting started
+
+`haskell.nix` can automatically translate your Cabal or Stack project and its dependencies into Nix code.
+
+This tutorial will teach you how to setup your project
+
+Assuming you have [Nix installed](https://nixos.org/download.html), you can start setting up your project.
+
+## Setting up the Cachix binary cache
```

Comment:
I think this should be in the main README too: we basically can't say this enough. This is the #1 thing where if  you don't do it you'll have a really bad time.

Reply:
The basic idea is that everyone should go through getting started. I confirmed that's what people were doing.

The reason noone did this was that setting up caches was always a step after installing nix-tools or building, which is way too late.

Reply:
There is a slightly related issue in that our primary cache is still hydra :-/

Reply:
@angerman not sure what exactly to do about that, I've copied what you had before, but let me know if we should change to something else.

I'm quite biased here, but Cachix is easier for people to setup.

Reply:
Certainly. Our Hercules instance feeds the cachix cache. Due to some inability to upgrade Hercules for a while it’s been falling behind. Hydra is the primary ci and feeds exclusively into the iohk hydra cache.